sky/engine/platform/fonts/apple/FontCacheIOS.mm

```diff
@@ -94,15 +94,22 @@
         CTFontCreateWithFontDescriptor(desc.get(), ui_desc.pointSize, nullptr)};
   }
 
-  CFRef<CTFontDescriptorRef> fallbackFont(unichar glyph) {
+  CFRef<CTFontDescriptorRef> fallbackFont(UChar32 glyph) {
     if (!_prototype) {
       return {};
     }
 
     base::mac::ScopedNSAutoreleasePool pool;
 
-    CFRef<CFStringRef> unicode_string(CFStringCreateWithFormat(
-        kCFAllocatorDefault, nullptr, CFSTR("%C"), glyph));
+    glyph = CFSwapInt32HostToLittle(glyph);
```

Comment:
I'm confused.  Is the passed in UChar32 not actually a UChar32 but rather two UChar16's which might form a surrogate pair?